scrapi/harvesters/elife.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*- t
+"""
+A harvester for metadata from eLife on Github for the SHARE project
+
+Sample API call:https://api.github.com/repos/elifesciences/elife-articles/commits?since=2015-12-22&until=2015-12-31&page=1&per_page=100
+"""
+
+from __future__ import unicode_literals
+
+import sys
+
+import logging
+import datetime
+
+from lxml import etree
+import json
+
+from itertools import chain
+
+from scrapi import requests
+from scrapi import settings
+from scrapi.base import XMLHarvester
+from scrapi.linter.document import RawDocument
+from scrapi.base.helpers import build_properties, compose, single_result, datetime_formatter
+
+logger = logging.getLogger(__name__)
+
+
+def collapse_list(list_of_strings):
+    text = ''.join(list_of_strings)
+    return text
+
+
+def elife_name_parser(names):
+    contributors = []
+    for i in range(0, len(names), 2):
+        chunka = names[i:i + 2]
+        chunkb = chunka[1].split(" ")
+        name = (chunka + chunkb)
+        del name[1]
+        contributors.append(name)
+
+    parsed_contributors = []
+    for contributor in contributors:
+        if sys.version_info < (3,):
+            contributor = map(lambda x: x.encode('utf-8'), contributor)
+
+        if len(contributor) == 3:
+            full_name = contributor[1] + str(" ") + contributor[2] + str(" ") + contributor[0]
+            first_name = contributor[1]
+            middle_name = contributor[2]
+            last_name = contributor[0]
+
+        else:
+            full_name = contributor[1] + str(" ") + contributor[0]
+            first_name = contributor[1]
+            middle_name = ""
+            last_name = contributor[0]
+
+        contributor_dict = {
+            'name': full_name,
+            'givenName': first_name,
+            'additionalName': middle_name,
+            'familyName': last_name
+        }
+
+        parsed_contributors.append(contributor_dict)
+
+    return parsed_contributors
+
+
+def elife_date_parser(date):
+    date_form = datetime.datetime(int(date[2]), int(date[1]), int(date[0]), 0, 0)
+    return date_form.date().isoformat()
+
+
+# will need to update this to pull from multiple pages
```

Comment:
If it's not hard, I'd prefer to have it working. We may want to go back that far at some point.